js/about/preferences.js

```diff
@@ -686,6 +686,21 @@ class PaymentsTab extends ImmutableComponent {
     </div>
   }
 
+  get nextReconcileDate () {
```

Comment:
this should return null if `ledgerData.get('reconcileStamp')` is not a number (may be undefined/null)